CommonAssemblyInfo.cs

```diff
@@ -29,7 +29,7 @@ Jeroen Frijters
 [assembly: AssemblyCopyright("Copyright (C) 2002-2011 Jeroen Frijters")]
 [assembly: AssemblyTrademark("")]
 [assembly: AssemblyCulture("")]
-[assembly: AssemblyVersion("0.47.0.1")]
+[assembly: AssemblyVersion("0.47.0.2")]
```

Comment:
Did you think about the impacts at build time and/or runtime?
If some of them, could you list them?

Reply:
At runtime we use assemblyBinding to redirect 0.47.0.2. It works
The idea is that we need a new version in order to distinguish that we have a new content.

However, still need to update this commit: 
- remove this file and update CommonAssemblyInfo.cs.in as that seems to be actual source of truth
- clarify why we have AssemblyFileVersion 0.47.1.0 and AssemblyVersion as 0.47.0.1 (typo or some other reason)